Makefile

```diff
@@ -16,7 +16,8 @@ GO_DEPENDENCIES = github.com/ory/go-acc \
 				  github.com/go-swagger/go-swagger/cmd/swagger \
 				  golang.org/x/tools/cmd/goimports \
 				  github.com/mattn/goveralls \
-				  github.com/cortesi/modd/cmd/modd
+				  github.com/cortesi/modd/cmd/modd \
+				  github.com/mailhog/MailHog
```

Comment:
Maybe a good idea to add this to `go_mod_indirect_pins.go`.

Reply:
Can do, could you explain, what that the point of that file is? I don't quite understand it.

Reply:
This pattern is used to manage/pin versions of tools the repo uses (like code generators, coverage calculators, testing utilities like MailHog) using the `go.mod` file of the actual Go module -- kratos in this case.

`go_mod_indirect_pins.go` is excluded from compilation via build tags which are never set. In fact, that file cannot be compiled at all. But it is part of the dependency resolution, and it's dependencies are noted down in `go.mod`. It serves only to import (as `_`) all the tools required in that repo, which are themselves written in Go.

Reply:
ah, I see. Thanks, I'll add Mailhog there as well.